### PR TITLE
🥒 Add scenarios for Integration Errors page

### PR DESCRIPTION
**Pull request by josemigallas**
image

### CODE REVIEW

**Review by lvillen — APPROVED** [review]

File: features/api/services/errors.feature [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  Scenario: List of errors
+    Given the product has the following integration errors:
+      | Timestamp | Message |
+      | 3/3/1991  | Error 1 |
+      | 4/4/1992  | Error 2 |
+    When they go to the integration errors page of product "Pepe API"
+    Then they should see the following table:
+      | Time (UTC)                | Error   |
+      | 1991-03-03 00:00:00 +0100 | Error 1 |
+      | 1992-04-04 00:00:00 +0200 | Error 2 |
```
Comment (akostadinov) [inline features/api/services/errors.feature:30]: is this gonna work on a machine with a different timezone? Where does timezone info come from?

File: features/api/services/errors.feature [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  Scenario: Empty list
+    Given the product has the following integration errors:
+      | Timestamp | Message |
+      | 3/3/1991  | Error 1 |
+      | 4/4/1992  | Error 2 |
+    When they go to the integration errors page of product "Pepe API"
+    And press "Purge"
+    And wait a moment
+    # TODO: Since ThreeScale::Core::ServiceError is mocked, errors are not really deleted.
+    # Then they should see "Hooray! No integration errors reported for this service."
```
Comment (akostadinov) [inline features/api/services/errors.feature:41]: is this supposed to stay like that for this PR?

**Comment by josemigallas** [comment]
Closing in favor of https://github.com/3scale/porta/pull/3765

File: features/api/services/errors.feature (hunk 2 above)
Comment (josemigallas) [reply features/api/services/errors.feature:41]: It was, but now it won't. Please take a look at #3765

**Pull request closed without merging**